src/components/settings/SettingsLMSTab/LMSConfigs/Blackboard/BlackboardConfig.tsx

```diff
@@ -3,12 +3,13 @@ import LmsApiService from "../../../../../data/services/LmsApiService";
 import { camelCaseDict, snakeCaseDict } from "../../../../../utils";
 import {
   BLACKBOARD_OAUTH_REDIRECT_URL,
+  BLACKBOARD_TYPE,
   LMS_CONFIG_OAUTH_POLLING_INTERVAL,
   LMS_CONFIG_OAUTH_POLLING_TIMEOUT,
   SUBMIT_TOAST_MESSAGE,
 } from "../../../data/constants";
 // @ts-ignore
-import BlackboardConfigActivatePage from "./BlackboardConfigActivatePage.tsx";
+import ConfigActivatePage from "../ConfigBasePages/ConfigActivatePage";
```

Comment:
Need to make sure and add .tsx file extension, or the build will fail.
```suggestion
import ConfigActivatePage from "../ConfigBasePages/ConfigActivatePage.tsx";
```